src/Entity/IdeasWorkshop/Answer.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace AppBundle\Entity\IdeasWorkshop;
+
+use AppBundle\Entity\Adherent;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+
+/**
+ * @ORM\Entity
+ *
+ * @Algolia\Index(autoIndex=false)
+ */
+class Answer
+{
+    /**
+     * @var int
+     *
+     * @ORM\Column(type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
```

Comment:
`strategy="AUTO"` is useless here, it is the default value of `GeneratedValue`

You can use directly : `@ORM\GeneratedValue`

Reply:
:heavy_check_mark: